supportedBackends/google/pipelines/v2alpha1/src/test/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesConversionsSpec.scala

```diff
@@ -25,8 +30,15 @@ class PipelinesConversionsSpec extends FlatSpec with Matchers {
       |""".stripMargin
   )
 
+  private lazy val fakeCredentials = NoCredentials.getInstance
+
+  private lazy val httpClientBuilder = HttpClientBuilder.create()
+
+  private def drsReadInterpreter(marthaResponse: MarthaResponse): IO[ReadableByteChannel] = ???
```

Comment:
Change `???` to `throw new UnsupportedOperationException`. The former signals that the entire JVM should be shutdown if this line is accidentally run.

Reply:
I must have missed this while changing other `???` to throw `UnsupportedOperationException`. Will change it now 👍